src/adagrams.js

```diff
@@ -1,8 +1,146 @@
-const Adagrams = {
+// reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/constructor
+
+class Adagrams {
+  constructor() {
+    this.lettersPool = {
+      A: 9, B: 2, C: 2, D: 4, E: 12, F: 2, G: 3, H: 2, I: 9, J: 1, K: 1, L: 4, M: 2, N: 6, O: 8, P: 2, Q: 1, R: 6, S: 4, T: 6, U: 4, V: 2, W: 2, X: 1, Y: 2, Z: 1
+    };
+
+    this.scoreChart = {
+      1: ["A", "E", "I", "O", "U", "L", "N", "R", "S", "T"],
+      2: ["D", "G"],
+      3: ["B", "C", "M", "P"],
+      4: ["F", "H", "V", "W", "Y"],
+      5: ["K"],
+      8: ["J", "X"],
+      10: ["Q, Z"]
+    };
+  };
+
+  // Implement this method for wave 1
   drawLetters() {
-    // Implement this method for wave 1
-  },
+    const hand = [];
+    const lettersPool = this.lettersPool;
+ 
+    for (const letter in lettersPool) {
+      for (let i = 0; i < lettersPool[letter]; i++) {
+        hand.push(letter);
+      };
+    };   
+
+    return this.sample(hand);
+  };
+
+
+  // Helper function for wave 1
+  // reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random
+  randomIdx(max) {
+    return Math.floor(Math.random() * Math.floor(max));
+  };
+
+
+  // Helper funciton for wave 1
+   sample(letters) {
```

Comment:
little thing: There's an extra space here. I use a VSCode plugin called indent-rainbow to help me see little things like this.